src/backend/firebase-backend.js

```diff
@@ -138,9 +138,14 @@ allItemsBought returns the IDs of all the items bought by a buyer
     returns: a promise containing an array of listing IDs
 */
 function allItemsBought(buyerID) {
+  console.log(buyerID,"me");
```

Comment:
If you want to leave these log statements in for debugging, make sure to comment them out when you commit the file so that they don't create noise in the console for anyone not using them.